CHANGELOG.md

```diff
@@ -1,5 +1,105 @@
+- `metro` packages are now at v0.54.1 ([7ff3874](https://github.com/facebook/react-native/commit/7ff3874), [343f0a1](https://github.com/facebook/react-native/commit/343f0a1) by [@motiz88](https://github.com/motiz88))
+- TODO add mention of AndroidX *BREAKING*
+
+#### Android specific
+
+- Move message queue initialization in ReactContext into its own method ([db40c28](https://github.com/facebook/react-native/commit/db40c28) by [@ejanzer](https://github.com/ejanzer))
+- Use class canonical name for `PARTIAL_WAKE_LOCK` tag ([88dbb45](https://github.com/facebook/react-native/commit/88dbb45) by [@timwangdev](https://github.com/timwangdev))
+
+#### iOS specific
+
+- *BREAKING*: Split React.podspec into separate podspecs for each Xcode project; your libraries will need to update for this change as well to avoid CocoaPods build errors ([2321b3f](https://github.com/facebook/react-native/commit/2321b3f) by [@fson](https://github.com/fson))
+- **SectionList**'s `scrollToLocation` on iOS now counts `itemIndex` like Android; both platforms are now consistent ([248a108](https://github.com/facebook/react-native/commit/248a108) by [@vonovak](https://github.com/vonovak))
+- Switch **Slider** `onSlidingComplete` event to a non-bubbling event on iOS to match Android ([7927437](https://github.com/facebook/react-native/commit/7927437) by [@rickhanlonii](https://github.com/rickhanlonii))
+
+### Deprecated
+
+- **StatusBar** is no longer deprecated; thank you for the feedback ([a203ebe](https://github.com/facebook/react-native/commit/a203ebe) by [@cpojer](https://github.com/cpojer))
+
+### Removed
+
+- **NetInfo** has been removed; its replacement is now available via the [react-native-community/netinfo](https://github.com/react-native-community/react-native-netinfo) package ([5a30c2a](https://github.com/facebook/react-native/commit/5a30c2a) by [@cpojer](https://github.com/cpojer))
+- **WebView** has been removed; its replacement is now available via the [react-native-community/webview](https://github.com/react-native-community/react-native-webview) package ([](https://github.com/facebook/react-native/commit/6ca438a), [1ca9a95](https://github.com/facebook/react-native/commit/1ca9a95), and [954f715](https://github.com/facebook/react-native/commit/954f715) by [@cpojer](https://github.com/cpojer) and [@thorbenprimke](https://github.com/thorbenprimke))
+- **Geolocation** has been removed; its replacement is now available via the [react-native-community/geolocation](https://github.com/react-native-community/react-native-geolocation) package ([17dbf98](https://github.com/facebook/react-native/commit/17dbf98) and [9834c58](https://github.com/facebook/react-native/commit/9834c58) by [@cpojer](https://github.com/cpojer) and [@mmmulani](https://github.com/mmmulani))
+- **AsyncStorage** no longer attempts to use RocksDB ([f854ce4](https://github.com/facebook/react-native/commit/f854ce4) by [@mmmulani](https://github.com/mmmulani))
```

Comment:
This feels more like an update than a removal to me. I also believe this logic doesn't impact open source, but actually brings FB more in line with open source. In that case this can probably just be removed.